utils.py

```diff
@@ -1096,7 +1096,60 @@ def maxpeak(G, w_start=-2, w_end=2, points=1000):
 #                                Chapter 3                                    #
 ###############################################################################
 
-
+def sym2mimotf(Gmat):
+    """Converts a MIMO transfer function system in sympy.Matrix form to a mimotf object making use of individual tf objects.
+    
+    Parameters 
+    ----------
+    Gmat : sympy matrix
+           The system transfer function matrix.
+    
+    Returns
+    -------
+    Gmimotf : sympy matrix
+              The mimotf system matrix
+        
+    Example
+    -------
+    >>> s=sympy.Symbol("s")
+    
+    >>> G=sympy.Matrix([[1/(s+1),1/(s+2)],
+    ...                 [1/(s+3),1/(s+4)]])
+    
+    >>> sym2mimotf(G)
+    mimotf([[tf([ 1.], [ 1.  1.]) tf([ 1.], [ 1.  2.])]
+     [tf([ 1.], [ 1.  3.]) tf([ 1.], [ 1.  4.])]])
+     
+    """
+    rows,cols=Gmat.shape
+    #create empty list of lists. This will be appended to form mimotf input list
+    Gtf=[[] for y in range(rows)]
+    
+    for i in range(rows):
```

Comment:
Avoid this kind of loop (`for in in range()`) Rather use something like `for row in Gmat; for G in row`